docs/src/integrations.md

```diff
@@ -0,0 +1,110 @@
+# Integrations
+
+Built on top of Apache Arrow, `LanceDB` is easy to integrate with the Python ecosystem, including Pandas, PyArrow and DuckDB.
+
+## Pandas and PyArrow
+
+``` sh
+pip install pandas lancedb
```

Comment:
this should be covered in the main section already.

Reply:
ok, removed.